ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -160,6 +160,11 @@ def write_tables(self, table_and_db: OMetaDatabaseAndTable):
                     table_id=created_table.id, sample_data=table_and_db.table.sampleData
                 )
             if table_and_db.table.tableProfile is not None:
+                for pd in table_and_db.table.tableProfile[0]:  # Only for entry 0?
```

Comment:
Not sure if this is the correct way to handle this

Reply:
it's better to loop here and fix it. Just in case if we get profiles for multiple days

Reply:
Fixed